interface/github.py

```diff
@@ -1,14 +1,17 @@
 """Utility classes for interacting with Github API via PyGithub."""
 from github import Github, GithubObject, GithubException
+from github.NamedUser import NamedUser
+from github.Team import Team
 from interface.exceptions.github import GithubAPIException
 from interface.github_app import GithubAppInterface, \
     DefaultGithubAppAuthFactory
-from github.Team import Team
-from typing import cast
+from typing import cast, List
+from functools import wraps
 
 
 def handle_github_error(func):
     """Github error handler that updates Github App API token if necessary."""
+    @wraps(func)
```

Comment:
What does `@wraps` do?

Reply:
It wraps the function so that the name and docstring of the function (re)surfaces in sphinx autodocs.